article/_posts/2021-07-29-DiSCo.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: review
```

Comment:
Maybe the "blog" layout fits better for this entry? Otherwise, some header fields specific to the "review" type are missing, and thus I guess the site would not be built correctly (?).

Reply:
Changed in d6aefbc0de10be4f97613718234cedae4bd20da6